cli/ops/io.rs

```diff
@@ -257,3 +259,23 @@ pub fn op_write(
   }
   .boxed_local()
 }
+
+#[derive(Deserialize)]
+struct CloseArgs {
+  rid: i32,
+}
+
+fn op_close(
+  state: &State,
+  args: Value,
+  _zero_copy: Option<ZeroCopyBuf>,
+) -> Result<JsonOp, OpError> {
```

Comment:
I think this should be in cli/ops/resources.rs ... what do you think?

Reply:
Not convinced - `resources` are baked into runtime. I was thinking about moving `op_resources` to `ops/runtime.rs`.

I'm open to any changes

Reply:
Close is also baked into the runtime - it’s how you drop a resource.